<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyFilterForm</class>
 <widget class="QWidget" name="MyFilterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Мастер фильтр</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resfile/src.qrc">
    <normaloff>:/svg/master_filter-icon.svg</normaloff>:/svg/master_filter-icon.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>7</number>
   </property>
   <property name="topMargin">
    <number>7</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="1" column="0">
    <widget class="QTreeView" name="treeView">
     <property name="styleSheet">
      <string notr="true">QAbstractItemView:item::selected{color:#fff;background-color: #4586D8;}

 QTreeView::branch:has-siblings:!adjoins-item {
	border-image: url(:/for_tree/stylesheet-vline.png) 0;
 }

 QTreeView::branch:has-siblings:adjoins-item {	
	border-image: url(:/for_tree/stylesheet-branch-more.png) 0;
 }

 QTreeView::branch:!has-children:!has-siblings:adjoins-item {	
	border-image: url(:/for_tree/stylesheet-branch-end.png) 0;
 }

 QTreeView::branch:has-children:!has-siblings:closed,
 QTreeView::branch:closed:has-children:has-siblings {
         border-image: none;	
	image: url(:/for_tree/stylesheet-branch-closed.svg);

 }

 QTreeView::branch:open:has-children:!has-siblings,
 QTreeView::branch:open:has-children:has-siblings  {
         border-image: none;         
	image: url(:/for_tree/stylesheet-branch-open.svg);
 }</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="autoExpandDelay">
      <number>1</number>
     </property>
     <property name="itemsExpandable">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="4">
       <widget class="QToolButton" name="clearBt">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resfile/src.qrc">
          <normaloff>:/svg/trash_.svg</normaloff>:/svg/trash_.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="addBt">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resfile/src.qrc">
          <normaloff>:/svg/add_filter_group_icon.svg</normaloff>:/svg/add_filter_group_icon.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="remBt">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resfile/src.qrc">
          <normaloff>:/svg/minus-icon.svg</normaloff>:/svg/minus-icon.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="addConditionBt">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resfile/src.qrc">
          <normaloff>:/svg/add_filter-icon.svg</normaloff>:/svg/add_filter-icon.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resfile/src.qrc"/>
 </resources>
 <connections/>
</ui>
